packages/nimble-components/src/table/models/keyboard-navigation-manager.ts

```diff
@@ -0,0 +1,1370 @@
+import { Notifier, Subscriber, Observable } from '@microsoft/fast-element';
+import {
```

Comment:
Think I have a general question of is there other state like the tabindex and the action menu focused state that the keyboard-navigation-manager is owning implicitly that I missed that can be made more explicit in the follow-on tech debt through more direct properties bound in templates?

Reply:
It's just tabindex and the action menu focused CSS class, so I think the other comments cover it.